doc/user/truffleruby-additions.md

```diff
@@ -20,9 +20,9 @@ In Ruby code, you can look at the standard `RUBY_ENGINE` constant, which will be
 `'truffleruby'`.
 
 It is also possible to use feature-detection instead of looking at
-`RUBY_ENGINE`. For example if you are writing an application that needs
-JavaScript interoperability you could test for `defined?(Truffle::Interop) &&
-Truffle::Interop.mime_type_supported?('application/javascript')`.
+`RUBY_ENGINE`. For example if you are writing an application that uses
+TruffleRuby's polyglot programming functionality you could test for
+`defined?(Polyglot)`.
```

Comment:
`Polyglot` will also be defined on SVM though, where e.g. Java interop does not work and in some launchers which might need `--polyglot` to allow accessing other languages.
Maybe `defined?(Truffle)` makes more sense?
There is also https://github.com/cjheath/polyglot so not sure the name Polyglot is very unique.

Reply:
The name may conflict, yes. It was decided on cross-language and we wanted it to be flatter than something like `Truffle::Interop`, and we want to downplay the `Truffle` name for user-visible branding. Discuss on Slack if you need to.

`defined?(Truffle)` isn't feature detection - that's engine detection. I should not define `Java` on native though - that's a good idea.